lib/index.js

```diff
@@ -183,6 +183,12 @@ function getPolyfillString(options) {
 					);
 				}
 
+				if ('all' in options.features) {
+					let warnText = 'Using the `all` alias with polyfill.io is a very bad idea. In a future version of the service, `all` will deliver the same behaviour as `default`, so we recommend using default instead.';
```

Comment:
We should create a page (on polyfill.io or Github wiki/issue) which explains why it is a _very bad idea_ to use the `all` alias. Thinking about it some more, maybe it should link to the [Github issue which proposed removing the feature](https://github.com/Financial-Times/polyfill-service/issues/655), that way users can comment on if they would like us to keep the feature around and what their use-cases are. An RFC from our users.